lizrd/scripts/run_exp_remotely.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# INSTRUCTIONS:
+# 1. needs to be called from sparsity folder
+# 2. needs to be called with the host as the first argument (as configured in ~/.ssh/config, e.g. gpu_entropy)
+# 3. needs to be called with the config file as the second argument (e.g. "my_config.yaml")
+# EXAMPLE USAGE: bash lizrd/scripts/run_exp_remotely.sh atena quick.json
+set -e
+
+source venv/bin/activate
+# run your python script
+python3 -m lizrd.scripts.sync_with_remote --host $1
+
+run_grid_remotely() {
+  host=$1
+  config=$2
+  script="cd ~/sparsity && find . -name $config -exec python3 -m lizrd.scripts.grid '{}' \;"
```

Comment:
1.`sparsity` -> `llm-random`
2. Dowiedzieć się od Tomka O. / zastanowić się czy nie ma jakiegoś downside'u z trzymaniem `llm-random` w $HOME na athenie. Nie wiem czy wszyscy, ale ja nie trzymam na athenie tego w $HOME tylko w `/net/pr2/projects/plgrid/plggllmeffi/mpioro/` i nie pamiętam z jakiego to powodu szczerza, ale wydaje mi się, że TO tak sugerował.
3. Nie jestem fanem rozwiązania z `findem`, ale może dlatego, że nie wiem jak `find` działa. ja mam wiele configów o tej samej nazwie, ale w różnych katalogach. Czy `find` tego nie psuje?

Reply:
1. Done, ludzie będą musieli przemianować swoje foldery tho lokalnie + na klastrze.
2. Huh, ok, dowiem się
3. Fair, to zakłada, że się zaczyna configa od daty tak jak ja. Dobra, to w takim razie hard ścieżka. Czy jakieś inne propozycje?